### ISSUE TITLE
[DISCUSS] README content and folder naming

### ISSUE THREAD

**Issue by echarles** [jtpio/jupyterlab-extension-tutorial#2]
Instead of having all the example explanation in the main README, I would rather see them moved to their dedicated folder.

Regarding the folder name, I would rather remove the prefix number. A the end, we can not order the examples by difficulty, only maybe by area

```
hello_world
commands
menus
datagrid
signals_and_buttons
kernel_messaging
kernel_output
file_server
widgets_...
widgets_...
```
@jtpio WDYT? After merge of #1 I can quickly open a PR for this.

**Comment by jtpio** [comment]
Thanks @echarles for the suggestion, that sounds good :+1:

We can always reintroduce an order later (a table of contents for instance) if we think it's needed.

**Comment by echarles** [comment]
Cool. If you merge #1 today, I can open a PR by tomorrow morning for this.

**Comment by jtpio** [comment]
@echarles you should have commit rights now, so feel free to merge #1.

**Comment by echarles** [comment]
@jtpio I have merge #1 and opened #3 PR for review. Thx.

**Comment by echarles** [comment]
Implemented with #3 Thx @jtpio for your reviews and last mile work.
